python/lsst/ts/externalscripts/base_make_calibrations.py

```diff
@@ -60,6 +62,46 @@ def __init__(self, index, descr):
 
         self.current_image_type = None
 
+        # Supported calibrations types
+        self.supported_calibrations_generation = [
```

Comment:
Instead of having both `self.supported_calibrations_generation` and `self.pipetask_parameters`, consider having only `self.pipetask_parameters`. Same for `self.supported_calibrations_verification` and `self.pipetask_parameters_verification`.